{"totalCount":3,"grade":"D","repositories":[{"url":"https://github.com/MTES-MCT/aides-territoires","vulnerabilityAlerts":{"totalCount":3,"nodes":[{"dismissedAt":null,"createdAt":"2022-09-26T18:39:17Z","securityVulnerability":{"severity":"HIGH","package":{"name":"scrapy"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-h7wm-ph43-c39p"},{"type":"CVE","value":"CVE-2017-14158"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2017-14158"},{"url":"https://github.com/scrapy/scrapy/issues/482"},{"url":"http://blog.csdn.net/wangtua/article/details/75228728"},{"url":"https://github.com/pypa/advisory-database/blob/8b7a4d62a95e8f605e5dfb4e0b4f299e6403dc12/vulns/scrapy/PYSEC-2017-83.yaml"},{"url":"https://github.com/advisories/GHSA-h7wm-ph43-c39p"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/scrapy/PYSEC-2017-83.yaml"}]}}},{"dismissedAt":null,"createdAt":"2022-10-18T18:48:45Z","securityVulnerability":{"severity":"HIGH","package":{"name":"django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-qrw5-5h28-6cmg"},{"type":"CVE","value":"CVE-2022-41323"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2022-41323"},{"url":"https://github.com/django/django/commit/5b6b257fa7ec37ff27965358800c67e2dd11c924"},{"url":"https://www.djangoproject.com/weblog/2022/oct/04/security-releases/"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2022-304.yaml"},{"url":"https://docs.djangoproject.com/en/4.0/releases/security/"},{"url":"https://groups.google.com/forum/#!forum/django-announce"},{"url":"https://github.com/advisories/GHSA-qrw5-5h28-6cmg"}]}}},{"dismissedAt":null,"createdAt":"2022-10-18T22:43:31Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"py"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-w596-4wvx-j9j6"},{"type":"CVE","value":"CVE-2022-42969"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2022-42969"},{"url":"https://github.com/pytest-dev/py/issues/287"},{"url":"https://github.com/pytest-dev/py/blob/cb87a83960523a2367d0f19226a73aed4ce4291d/py/_path/svnurl.py#L316"},{"url":"https://github.com/pytest-dev/py/issues/288"},{"url":"https://github.com/pytest-dev/pytest/issues/10392"},{"url":"https://github.com/advisories/GHSA-w596-4wvx-j9j6"}]}}}]},"grade":"D"}]}